live-examples/js-examples/promise/promise-resolve.html

```diff
@@ -1,9 +1,14 @@
 <pre>
-<code id="static-js">var promise1 = Promise.resolve([1, 2, 3]);
+<code id="static-js">var promise1 = Promise.resolve(123);
 
 promise1.then(function(value) {
   console.log(value);
-  // expected output: Array [1, 2, 3]
+  // expected output: 123
 });
+
+Promise.resolve(42)
+  .then(console.log);
+// expected output: 42
```

Comment:
I reckon we can remove these last three lines. It might confuse especially newcomers who will then focus more on understanding this syntax than on `Promise.resolve`. So in the end the example should be just:

```
var promise1 = Promise.resolve(123);

promise1.then(function(value) {
  console.log(value);
  // expected output: 123
});
```